dts/bindings/adc/shunt-sensor.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 The ChromiumOS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Description for a current sensor measured using a shunt resistor and
+    parallel voltmeter.
+
+compatible: "shunt-sensor"
```

Comment:
Can you work on a separate PR for this?  Keeping things isolate to help out the review process.